k2/torch/csrc/fsa_class.cu

```diff
@@ -178,8 +176,8 @@ RaggedArc RaggedArc::FromBinaryFunctionTensor(const RaggedArc &a_src,
   return dest;
 }
 
-void RaggedArc::CopyTensorAttrs(const RaggedArc &src, torch::Tensor arc_map,
-                                bool over_write /*= true*/) {
+void FsaClass::CopyTensorAttrs(const FsaClass &src, torch::Tensor arc_map,
+                               bool over_write /*= true*/) {
   for (const auto &iter : src.tensor_attrs) {
     if (over_write || !HasAttr(iter.first)) {
       float filler = GetFiller(iter.first);
```

Comment:
In line 187, it calls the overload of `SetAttr` accepting tensor arguments, but we atually want to use the overload using `IValue`.

Please change the name of `SetAttr` that takes tensor arguments to `SetTensorAttr`.